composer.json

```diff
@@ -20,8 +21,8 @@
         ],
         "psr-4": {
             "App\\": "app/"
-        }
-       
+        },
+        "files": ["app/Helper.php"]
```

Comment:
do it need an autoloading? "files with namespace App\ are autoloaded" - @sadhakbj